test/file_sqlite/setup_sqlite.sql

```diff
@@ -0,0 +1,28 @@
+DROP TABLE IF EXISTS "test_endpoints";
+
+CREATE TABLE "test_endpoints" (
+    "endpoint" TEXT NOT NULL,
+    "method" TEXT NOT NULL DEFAULT 'GET',
+    "file" TEXT,
```

Comment:
Ich finde es besser, wenn `testdata.sqlite` direkt im test liegen und nur shared resources in `_res`.